pkg/cluster/cluster.go

```diff
@@ -471,7 +471,13 @@ func GetClusterAddOns(connection *sdk.Connection, clusterID string) ([]*AddOnIte
 
 			// Only display add-ons for which the org has quota
 			resourceQuotas.Each(func(resourceQuota *amsv1.ResourceQuota) bool {
-				if addOn.ResourceName() == resourceQuota.ResourceName() {
+				sku := resourceQuota.SKU()
+				skuRuleResponse, err := connection.AccountsMgmt().V1().SkuRules().SkuRule(sku).Get().Send()
```

Comment:
you should query for all SkuRules once and keep a list.  You can put them in a map by id for fast lookup.

This implementation will make a separate API call for each resourceQuota object.  This is far less efficient than making 1 query.

Reply:
and you’ll have many SkuRules for a single ResourceQuota.QuotaID.  you can do a direct sku.QuotaID==rq.QuotaID comparison